package.json

```diff
@@ -35,12 +39,22 @@
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "ava": "^0.25.0",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.6.0",
+    "babel-preset-env": "^1.7.0",
```

Comment:
all of this is the old babel; the new stuff is @babel/* as you can see on line 40 here